### DISCUSSION TITLE
Cannot find auth0/ when using devtools

### DISCUSSION THREAD

**Discussion by classic-DSW** [drizzle-mizzle/CharacterAI-Discord-Bot#139] [Q&A] [answered]
Hello! I recently got everything i've needed for this, but for some reason it won't let me access auth0 when i open devtools. Is there any other way to access the user token if this is not available? thank you :[
![cries myself to sleep](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/128752380/948182c3-2898-4974-a81d-ebcc152c13fa)

**Comment by SanityMinds** [comment, marked as answer]
Hi, Not the repo owner. But a frequent user. I just noticed this now. 
I found where they now store the C.AI tokens 
See image! 

go to: Application > local storage > beta.character.ai or plus.character.ai > CHAR_TOKEN
![image](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/138660341/f8de9bad-991f-4b05-87e6-ede11c8b5982)

**Reply by SanityMinds** [reply]
yes I did expose my own token. Theres a bug on my account making the AI not respond to me.

**Reply by classic-DSW** [reply]
Holy shit thank you so much you have no idea how much i appreciate this

**Reply by drizzle-mizzle** [reply]
I'm too lazy to rewrite the steps and make screenshots, so I have simply linked your answer in [the guide](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/wiki/How-to-set-up#5-sign-in-open-devtools-ctrlshiftj-in-chrome-and-find-your-auth-token-as-it-described-here)

**Reply by drizzle-mizzle** [reply]
![image](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/assets/55811932/6704c278-c1d0-4359-abee-d98f97a666ff)

**Reply by ShieldVA** [reply]
I have an issue. My browser is not showing what is in your screenshot. I'm on Opera GX. I followed your steps, but I am not finding my token.
![image_2024-07-17_115454033](https://github.com/user-attachments/assets/8c87b3a1-420d-4584-b93d-ea9b8a092c0d)

**Comment by JustVrit** [comment]
What if bot writing in console Disconnected

**Reply by SanityMinds** [reply]
This project is no longer supported. But I suggest checking if tokens are correct (discord & C.AI) and would double check if everything is correct. 

I'd hope this project does come back. I personally don't like the new project and like the more Simplistic one (this one) although it is very buggy I use it for my own community. 

I suggest reaching out to @drizzle-mizzle anyways since he has been a big help in the past.

**Reply by drizzle-mizzle** [reply]
It will not just say Disconnected, it might display some error message. Either token or internet connection problem.
